crates/extensions/tedge_file_system_ext/src/lib.rs

```diff
@@ -121,9 +121,9 @@ impl Actor for FsWatchActor {
     }
 
     async fn run(&mut self) -> Result<(), RuntimeError> {
-        let mut fs_notify = NotifyStream::try_default().unwrap();
+        let mut fs_notify = NotifyStream::try_default()?;
         for (watch_path, _) in self.messages.get_watch_dirs().iter() {
-            fs_notify.add_watcher(watch_path).unwrap();
+            fs_notify.add_watcher(watch_path)?;
```

Comment:
In order to avoid adding a new `RuntimeError` case, you can simply box the error.

```suggestion
            fs_notify.add_watcher(watch_path).map_err(Box::new)?;
```

Reply:
addressed